wgpu-hal/src/gles/adapter.rs

```diff
@@ -298,7 +296,11 @@ impl super::Adapter {
                     .unwrap(),
                 },
             },
-        })
+        };
+
+        log::debug!("{:?}", adapter);
```

Comment:
If we want to print out the adapter, I think wgpu-core is a better place than here.

Reply:
Hmm, I think `Capabilities` is enough. No needed to print whole adapter. Can we print it here?

Reply:
Capabilities are also not GL specific, so any printing of them should be in wgpu-core

Reply:
But wgpu-core will print only for selected adapter? (Vulkan or GL, for example). Should we print this info for all backend specific adapters?

Reply:
I don't think so. Is this useful to print this on non-selected adapters?

Reply:
This might help with issue like this one #1590

Reply:
This one happens on device request, where we are selecting the exact adapter. So printing out the info at that point is perfectly fine.

Reply:
I removed this part from PR, will look at how we can implement this in wgpu-core.